### PR TITLE
Adds missing type information to Client

### PR DESCRIPTION
**Pull request by douglance**

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -1270,15 +1270,8 @@ export class Client {
       // Finally, serialize the transaction
       const serializedTx = bitcoin.serializeTx(preSerializedData);
       // Generate the transaction hash so the user can look this transaction up later
-      let preImageTxHash = serializedTx;
-      if (preSerializedData.isSegwitSpend === true) {
```
Comment (douglance) [inline src/client.ts:1274]: I went ahead and removed this block, since it was impossible to be executed.

**Comment by alex-miller-0** [comment]
#258 will collide with this. Can we get that one in first?

File: src/client.ts (hunk 1 above)
Comment (alex-miller-0) [reply src/client.ts:1318]: Good catch

**Review by alex-miller-0 — APPROVED** [review]
Looks good. Please just squash the merge commits.

**Pull request merged**